apps/src/redux/instructions.js

```diff
@@ -0,0 +1,76 @@
+var _ = require('../lodash');
+
+var SET_INSTRUCTIONS_IN_TOP_PANE = 'instructions/SET_INSTRUCTIONS_IN_TOP_PANE';
+var TOGGLE_INSTRUCTIONS_COLLAPSED = 'instructions/TOGGLE_INSTRUCTIONS_COLLAPSED';
+var SET_INSTRUCTIONS_HEIGHT = 'instructions/SET_INSTRUCTIONS_HEIGHT';
+var SET_INSTRUCTIONS_MAX_HEIGHT = 'instructions/SET_INSTRUCTIONS_MAX_HEIGHT';
```

Comment:
:+1: I'm liking this move from 'enums' to the namespaced constants that we don't need to export.